README.md

```diff
@@ -1,18 +1,77 @@
+[![Published on NPM](https://img.shields.io/npm/v/@polymer/iron-menu-behavior.svg)](https://www.npmjs.com/package/@polymer/iron-menu-behavior)
 [![Build status](https://travis-ci.org/PolymerElements/iron-menu-behavior.svg?branch=master)](https://travis-ci.org/PolymerElements/iron-menu-behavior)
+[![Published on webcomponents.org](https://img.shields.io/badge/webcomponents.org-published-blue.svg)](https://webcomponents.org/element/@polymer/iron-menu-behavior)
 
-_[Demo and API docs](https://elements.polymer-project.org/elements/iron-menu-behavior)_
+## `IronMenuBehavior`, `IronMenubarBehavior`
 
+`IronMenuBehavior` and `IronMenubarBehavior` implement accessible menu and
+menubar behaviors.
 
-## Polymer.IronMenuBehavior
+See: [Documentation](https://www.webcomponents.org/element/@polymer/iron-menu-behavior),
+  [Demo](https://www.webcomponents.org/element/@polymer/iron-menu-behavior/demo/demo/index.html).
 
-`Polymer.IronMenuBehavior` implements accessible menu behavior.
+## Usage
 
-## Polymer.IronMenubarBehavior
+### Installation
 
-`Polymer.IronMenubarBehavior` implements accessible menubar behavior.
+```
+npm install --save @polymer/iron-menu-behavior
+```
 
-### Notable breaking changes between 1.x and 2.x (hybrid):
+### In a Polymer 3 element
 
-IronSelectableBehavior and IronMultiSelectableBehavior, which are used by
-IronMenuBehavior and IronMenubarBehavior, introduce multiple breaking changes.
-Please see the README for those behaviors for more detail.
+```js
+import {PolymerElement} from '@polymer/polymer/polymer-element.js';
+import {mixinBehaviors} from '@polymer/polymer/lib/legacy/class.js';
+import {html} from '@polymer/polymer/lib/utils/html-tag.js';
+
+import {IronMenuBehavior} from '@polymer/iron-menu-behavior/iron-menu-behavior.js';
+
+class SimpleMenu extends mixinBehaviors(IronMenuBehavior, PolymerElement) {
```

Comment:
Can you also have a "using this element in an html page" example after this? It's not clear how to use this element (what should go in the slots, etc)....